docs/scripts/update-all-docs.sh

```diff
@@ -64,6 +73,9 @@ echo ".. _generic_easyblocks:" > $VERSION_SPECIFIC_DIR/generic_easyblocks.rst
 echo "" >> $VERSION_SPECIFIC_DIR/generic_easyblocks.rst
 python $generic_easyblocks_script >> $VERSION_SPECIFIC_DIR/generic_easyblocks.rst
 
+echo ":ref:\`Generic EasyBlocks <generic_easyblocks>\`" >> $overview
```

Comment:
just use
```
:ref:`generic_easyblocks`
```

that way the page title will be inherited

and make this a proper list, using `* :ref:...`